tests/Traits/JsonControllerTestable.php

```diff
@@ -15,10 +15,7 @@
 use function json_decode;
 use function substr;
 
-/**
- * Class JsonControllerTest
- */
-trait JsonControllerTest
+trait JsonControllerTestable
```

Comment:
Maybe being weird, but I don't love this name... maybe `TestableJsonController`?

Reply:
you go it.

:white_check_mark: